presto-main/src/test/java/io/prestosql/sql/planner/TestLogicalPlanner.java

```diff
@@ -943,41 +943,36 @@ public void testOffset()
     @Test
     public void testRedundantLimitNodeRemoval()
     {
-        String query = "SELECT count(*) FROM orders LIMIT 10";
```

Comment:
This one seems useful since we use it for both the assertion and the error message. Duplicating it means they might be different.